operator/api/v1alpha1/central_types.go

```diff
@@ -470,14 +470,66 @@ type ExposureNodePort struct {
 
 // ExposureRoute defines settings for exposing central via a Route.
 type ExposureRoute struct {
+	// Expose central with a passthrough route.
+	// The passthrough route is still required when using a reencrypt route.
 	//+kubebuilder:default=false
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=1
 	Enabled *bool `json:"enabled,omitempty"`
 
 	// Specify a custom hostname for the central route.
-	// If unspecified, an appropriate default value will be automatically chosen by OpenShift route operator.
+	// If unspecified, an appropriate default value will be automatically chosen by the OpenShift route operator.
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=2
 	Host *string `json:"host,omitempty"`
+
+	// Set up a central route with reencrypt tls termination.
+	// For reencrypt routes, the request is terminated on the OpenShift router with a custom certificate.
+	// The request is then reencrypted by the OpenShift router and sent to central.
+	// [user] --TLS--> [OpenShift router] --TLS--> [central]
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=3,displayName="Re-Encrypt Route"
+	Reencrypt *ExposureRouteReencrypt `json:"reencrypt,omitempty"`
+}
+
+// ExposureRouteReencrypt defines settings for exposing central via a reencrypt Route.
+type ExposureRouteReencrypt struct {
+	// Expose central with a reencrypt route.
+	// Requires a passthrough route for sensor communication.
```

Comment:
Hm, in theory sensor could be using yet another way to talk to central, but this sentence sounds as if a passthrough route is needed to use this re-encrypt route. Maybe just say that this reencrypt one is not to be used for sensor? 🤔 

OTOH I'm starting to doubt that sensor would be unable to use a reencrypt route. It would be a waste of CPU, but I think sensor has a way of traversing a proxy using an unfamiliar TLS cert...

Reply:
I've adjusted the comment to say that the reencrypt route should not be used for sensor. At least out of the box it won't work.